CHANGELOG.md

```diff
@@ -28,96 +77,83 @@ This means that druid no longer requires cairo on macOS and uses Core Graphics i
 
 ### Added
 
+- Standardized and exposed more methods on more contexts. ([#970], [#972], [#855] by [@cmyr], [#898] by [@finnerale], [#954] by [@xStrom], [#917] by [@jneem])
+- `im` feature, with `Data` support for the [`im` crate](https://docs.rs/im/) collections. ([#924] by [@cmyr])
+- `im::Vector` support for the `List` widget. ([#940] by [@xStrom])
 - `TextBox` can receive `EditAction` commands. ([#814] by [@cmyr])
 - `Split::min_splitter_area(f64)` to add padding around the splitter bar. ([#738] by [@xStrom])
 - Published `druid::text` module. ([#816] by [@cmyr])
 - `InternalEvent::MouseLeave` signalling the cursor left the window. ([#821] by [@teddemunnik])
+- `InternalEvent::RouteTimer` to route timer events. ([#831] by [@sjoshid])
 - `children_changed` now always includes layout and paint request. ([#839] by [@xStrom])
-- `UpdateCtx::submit_command`. ([#855] by [@cmyr])
 - `request_paint_rect` for partial invalidation. ([#817] by [@jneem])
 - Window title can be any `LabelText` (such as a simple `String`). ([#869] by [@cmyr])
 - `Label::with_font` and `set_font`. ([#785] by [@thecodewarrior])
-- `InternalEvent::RouteTimer` to route timer events. ([#831] by [@sjoshid])
 - `MouseEvent` now has a `focus` field which is `true` with window focusing left clicks on macOS. ([#842] by [@xStrom])
 - `MouseButtons` to `MouseEvent` to track which buttons are being held down during an event. ([#843] by [@xStrom])
 - `Env` and `Key` gained methods for inspecting an `Env` at runtime ([#880] by [@Zarenor])
-- `UpdateCtx::request_timer` and `UpdateCtx::request_anim_frame`. ([#898] by [@finnerale])
-- `LifeCycleCtx::request_timer`. ([#954] by [@xStrom])
 - `scale` method to `WinHandler`. ([#904] by [@xStrom])
 - `WinHandler::scale` method to inform of scale changes. ([#904] by [@xStrom])
```

Comment:
Those seem to be the same.

Reply:
Good catch. Some sort of rebase failure of the original PR I guess. I can't even see where the shorter line exactly got added.